src/test/java/edu/indiana/dlib/amppd/controller/AmpUserControllerTests.java

```diff
@@ -194,7 +195,17 @@ public void testForgotPasswordEmail() throws Exception {
     		       .content(json)
     		       .accept(MediaType.APPLICATION_JSON)).andExpect(status().isOk()).andExpect(jsonPath("$.success").isBoolean()).andExpect(jsonPath("$.success").value(true));
     }
-
+    
+    @Test
+    public void shouldTestGetUser() throws Exception {
+    	Random rand = new Random(); 
```

Comment:
Better call it shouldGetUserWithValidId if you are testing only valid ID.

Also, if you are getting user by random ID, won't that fail most time? what results did you get on this test?